website/orders/templates/orders/shift_overview.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'tosti/base.html' %}
+{% load static %}
+
+{% block styles %}
+    <link rel="stylesheet" href="{% static 'tosti/css/index.css' %}"/>
+    <link rel="stylesheet" href="{% static 'orders/css/forms.css' %}"/>
+    <link rel="stylesheet" href="{% static 'orders/css/item-list-admin.css' %}"/>
+{% endblock %}
+
+{% block page %}
+    <div class="container-sm">
+        <h1>{{ shift.venue.name }}</h1>
+        <h3>
+            shift #{{ shift.pk }}
+            {% if shift.assignees %}
+                -
+                {% for assignee in shift.get_assignees %}
+                    {{ assignee.get_short_name }}{% if not forloop.last %}, {% endif %}
+                {% endfor %}
+            {% else %}
+             - No assignees
+            {% endif %}
+        </h3>
+        <p>Shift open from {{ shift.start_date }} until {{ shift.end_date }}</p>
+    </div>
+    {% if user.is_staff %}
+        <div class="container-sm">
+            <div class="btn-ml btn-on bg-primary border-0 text-center" onclick="window.location.href = '{% url "orders:shift_admin" shift=shift %}';">
+                <h5>Admin view</h5>
+                <i class="fas fa-tools"></i>
+            </div>
+        </div>
+    {% endif %}
+    <div class="container-sm">
+        <h1>Orders</h1>
```

Comment:
On my machine, the orders don't load...?

Reply:
Deleted the javascript file on accident, it is back now.